docs/medical-api/getting-started/quickstart.mdx

```diff
@@ -219,3 +219,17 @@ That's it! You're now well equiped to use the Metriport API to access and manage
 
 As next steps, we'd recommend scoping out our [Medical API Reference](/medical-api/api-reference/organization/create-organization)
 in further detail to see the other things you can do with the API.
```

Comment:
need to put this section (and the line at 218) to the bottom now

and hyper nit: move the party emojis to step 8? 😆